test/mpl_interop_test1.cpp

```diff
@@ -5,26 +5,29 @@
 //  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
 #include <boost/type_traits/is_void.hpp>
+#include <boost/mpl/bool.hpp>
+#include <boost/config.hpp>
 
 template <class T>
-int dispatch_test_imp(const boost::mpl::bool_<true>&)
+BOOST_CONSTEXPR int dispatch_test_imp(boost::mpl::true_)
```

Comment:
Why are you changing const ref to value here? Does the original no longer work?

Reply:
I do not actually remember already, most likely to show that it is copyable at compile time.

Reply:
Reverted it back.

Reply:
Your modified test was valuable, just add it without removing the old one. Removing an old test implies that it no longer works as a result of a change, which isn't the case here.

Reply:
Agreed